### PR TITLE
Migrate the nvtRatio API to use the clickhouse v2 metrics

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: test/sanbase_web/graphql/clickhouse/nvt_api_test.exs [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
+         end
+       ]}
+    ]) do
       response = execute_query(context)
-      ratios = parse_response(response)
 
-      assert_called(NVT.nvt_ratio(context.slug, context.from, context.to, context.interval))
+      ratios =
+        parse_response(response)
+        |> IO.inspect(label: "60", limit: :infinity)
```
Comment (tspenov) [inline test/sanbase_web/graphql/clickhouse/nvt_api_test.exs]: Inspect left

File: lib/sanbase_web/graphql/resolvers/clickhouse_resolver.ex [hunk 2]
```diff
@@ -261,15 +260,33 @@ defmodule SanbaseWeb.Graphql.Resolvers.ClickhouseResolver do
...
+        |> Enum.map(fn {%{datetime: datetime, value: nvt_ratio_circulation},
+                        %{value: nvt_transaction_volume}} ->
+          %{
+            datetime: datetime,
+            nvt_ratio_circulation: nvt_ratio_circulation,
+            nvt_ratio_tx_volume: nvt_transaction_volume
+          }
+        end)
+
+      {:ok, result}
```
Comment (tspenov) [inline lib/sanbase_web/graphql/resolvers/clickhouse_resolver.ex]: We need logging/error handling for the else part of the `with` ?

Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/resolvers/clickhouse_resolver.ex]: I'll add it to the metric_resolver, so it's automatically handled for all places.

File: test/sanbase_web/graphql/clickhouse/nvt_api_test.exs [hunk 3]
```diff
@@ -1,110 +1,108 @@
...
+    ]) do
       response = execute_query(context)
       ratios = parse_response(response)
 
-      assert_called(NVT.nvt_ratio(context.slug, context.from, context.to, context.interval))
       assert ratios == []
     end
   end
 
-  test "logs warning when calculation errors", context do
```
[the remark this answers is not in the document]
Comment (IvanIvanoff) [reply test/sanbase_web/graphql/clickhouse/nvt_api_test.exs]: Fixed

**Pull request merged**